picasso/build.gradle

```diff
@@ -21,26 +20,20 @@ android {
   }
 
   dexOptions {
-    preDexLibraries = !rootProject.ext.isCi
-  }
-
-  defaultConfig {
-    minSdkVersion rootProject.ext.minSdkVersion
-
-    consumerProguardFiles 'consumer-proguard-rules.txt'
+    preDexLibraries = !isCi
   }
 }
 
 dependencies {
-  api dep.okhttp
-  api dep.supportAnnotations
-  implementation dep.supportExifInterface
-  testImplementation dep.junit
-  testImplementation dep.truth
-  testImplementation dep.supportV4
-  testImplementation dep.robolectric
-  testImplementation dep.mockito
-  testImplementation dep.mockWebServer
+  api deps.okhttp
+  compileOnly deps.supportAnnotations
```

Comment:
Why this and not `implementation`?

Reply:
the annotations aren't needed for runtime, so figured why not remove it from the runtime classpath?

Reply:
They're transitive of the exif library already. Without that, in the insane case a user doesn't have any other dependencies pulling annotations (basically impossible) it would cause ProGuard to fail. I would just leave it as implementation.